js/lib/beautify-html.js

```diff
@@ -84,7 +84,7 @@
         return s.replace(/\s+$/g, '');
     }
 
-    function style_html(html_source, options, js_beautify, css_beautify) {
+    function style_html(html_source, options, js_beautify_raw, css_beautify_raw) {
```

Comment:
But more accurately, I think you'd do better to change each beautifier so each on reads options + options.<language>.  Other wise you only solve this for the case where people are running the html beautifier.